### ISSUE TITLE
Username/Repository# References In Commits Displayed Incorrectly

### ISSUE THREAD

**Issue by dsanders11** [desktop/desktop#8403]
## Description

Autolinked references in commit messages of the form `Username/Repository#` are not displayed correctly in the app, but are on GitHub's website.
## Version

* GitHub Desktop: 2.1.3

* Operating system: 10.14.6

## Steps to Reproduce

1. Make a commit which references an issue on another repository, using the form `Username/Repository#`, e.g. "Testing desktop/desktop#9999999"

### Expected Behavior

Should show a clickable link in the repo history to the issue/PR on the **other** repository.

### Actual Behavior

Shows a clickable link in the repo history to the issue/PR on the **same** repository.

**Comment by tierninho** [comment]
@dsanders11 Thanks for the submission. Currently the logic is only intended to link to the current repository. We will consider your request in future iterations.

Until then, you may use the full URL if you like a link to appear, like so:

![Screen Shot 2019-10-04 at 7 45 58 AM](https://user-images.githubusercontent.com/14828183/66228475-3ff69000-e67b-11e9-9b8d-d6fd927d7bd7.png)

**Comment by dsanders11** [comment]
@tierninho, I still consider this a bug, regardless of the intended logic. Viewing the commit on GitHub vs viewing it in the app results in two different links, which can easily cause confusion.

**Comment by tierninho** [comment]
I agree with you as the consistency between Desktop and GitHub.com are ≠. Filing this as a bug.

**Comment by thinkyhead** [comment]
This was bugging me too, so I submitted a PR to fix it.

**Comment by tidy-dev** [comment]
See https://github.com/desktop/desktop/pull/15139 for direction.
